lib/src/showcase.dart

```diff
@@ -295,13 +300,6 @@ class _ShowcaseState extends State<Showcase> {
                 ),
               ),
               if (_isScrollRunning) Center(child: widget.scrollLoadingWidget),
-              if (!_isScrollRunning)
-                _TargetWidget(
```

Comment:
what is the reason for removing this widget ?

Reply:
This widget is used in the stack so the child gesture of overlay is not detectable over _TargetWidget's gesture.
So I have to use the onTargetTap action in the child of the AnchoredOverlay widget of the build method.